roles/backend_setup/tasks/main.yml

```diff
@@ -10,7 +18,7 @@
   package:
     name: device-mapper-persistent-data
     state: present
-  when: ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux'
+  when: ansible_distribution == 'CentOS' or ansible_distribution == 'RedHat'
```

Comment:
Can't you just use ansible_os_family ?

Reply:
@mykaul yep, we can reduce an extra comparison to CentOS.

Reply:
Additionally we are covered for Fedora as well. Thanks.